synapse/storage/databases/main/end_to_end_keys.py

```diff
@@ -1302,6 +1237,99 @@ async def claim_e2e_fallback_keys(
 
         return results
 
+    @trace
+    def _claim_e2e_one_time_key_simple(
```

Comment:
I'm assuming this is identical to the previous inner function `_claim_e2e_one_time_key_simple`.